### ISSUE TITLE
ZingTouch does not work with iOS 13

### ISSUE THREAD

**Issue by kmgwalker** [zingchart/zingtouch#70]
The demo page is unresponsive with ios 13 (ipad and iphone).  Works fine with ios 12 (ipad).

**Comment by Xerotherm1c** [comment]
Seems like this should be a top priority bug to fix - the latest iOS not working?? I found that hammer.js has some feature conflicts with iOS13 (documented [here](https://github.com/hammerjs/hammer.js/issues/1237)). One of the comments reads:

> Try to turn off the following setting
> 

> > Settings -> Safari -> Advanced -> Experimental Features -> Pointer Events

> 
> iOS 13 supports Pointer Event, so HammerJS seems to recognize Pointer Event instead of Touch.
> 

@kmgwalker Can you (or anyone with iOS13) test the ZingTouch examples with this setting turned off? I do not have an iOS13 device handy.

**Comment by whoacowboy** [comment]
I agree, this is pretty big. I started to look into it, but got side tracked with work. 

You can test on a Mac with a trackpad and safari as well.

This is what I came across as a starting point. 

```
function zoom(e) {
  console.log(e.scale)
  e.preventDefault()
}
document.addEventListener('gesturestart', zoom)
document.addEventListener('gesturechange', zoom)
document.addEventListener('gestureend', zoom)
```
I am goin to look into it again, once I have some more time. Any insights you find or PRs would be great.

**Comment by Xerotherm1c** [comment]
I was able to get my hands on an iPhone 7 running iOS 13.3.1 and the specific code example that does not work is the rotation example. Every other example works on this device, though the pinch-and-zoom graphing example appears to require focus before pinching-and-zooming.

I confirmed that disabling Pointer Events inside Safari's Experimental Settings fixes the otherwise-broken rotation example with both one or two fingers. I am unsure if this will likewise fix any otherwise-broken functionality in hammer.js, but given that I picked that tip up from a reported hammer.js issue, it's likely that this setting is the sole reason these libraries are (partially) broken in iOS13. Hopefully additional developers can cross-confirm these results and test other device models with various iOS13 minor versions, as I no longer have access to the iOS13 device.

**Comment by kmgwalker** [comment]
iOS 13.3.1, iphone 11:  On the demos on https://zingchart.github.io/zingtouch/ , (1) 1 and 2-finger taps work, (2) rotate does not work, (3) pinch/expand does NOT work (one must distinguish expanding the chart from the mobile-safari-built-in expansion of the entire web page), and (4) flicking bubbles does not work (one can make a bubble hold still, but one cannot send it in a new direction).

** With point events turned off (as described above), all of the examples work

**Comment by kylebakerio** [comment]
any news on this?
